src/Hazelcast.Net.Tests/Networking/RegressionWithRealNetworkTest.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Linq;
+using Hazelcast.Core;
+using Hazelcast.Networking;
+using Hazelcast.Testing;
+using Hazelcast.Testing.Logging;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+
+namespace Hazelcast.Tests.Networking
+{
+    public class RegressionWithRealNetworkTest : MultiMembersRemoteTestBase
+    {
+        private IDisposable HConsoleForTest()
+            => HConsole.Capture(options => options
+                .ClearAll()
+                .Configure<HConsoleLoggerProvider>().SetPrefix("LOG").SetMaxLevel()
+                .Configure().SetMinLevel().EnableTimeStamp(origin: DateTime.Now)
+                .Configure(this).SetMaxLevel().SetPrefix("TEST")
+            );
+
+        [TearDown]
+        public async Task TearDown()
+        {
+            //since tests are network related may need fresh members on cases.
```

Comment:
that base method does not remove members from `RcMembers` so you'll need to either fix the base method (add an `RcMembers.Clear()` of some sort, I guess...) or code a variant here that *does* clear `RcMembers`.

Reply:
Fixed at 1994fec7c1ec5775d246230ddb9ab7f388640b4b